pyproject.toml

```diff
@@ -50,6 +51,9 @@ path = "kolibri2zim/__about__.py"
 exclude = ["/.github"]
 
 [tool.hatch.build.hooks.custom]
+enable-by-default = false
```

Comment:
I guess you're adding this to prevent re-fetching the dependencies but you know this is not the proper solution to this problem… You can add a simple check for a known assets and skip the subprocess if present (we'll have to replace that sh script someday)

Reply:
I duplicated the list of expected dependencies, in order to try make it more clear that both list must be maintained. Otherwise I'm pretty sure that the asset I will choose will be removed in the future and no-one will understand what is broken.